include/tvm/runtime/object.h

```diff
@@ -1,877 +0,0 @@
-/*
```

Comment:
Why this file is deleted?

Reply:
It is deleted by mistake. I will update the PR soon. Thank you for your help!